### PR TITLE
Rename "machine learning tool" to "AI creator"

### PR DESCRIPTION
**Pull request by microbit-grace**
Includes renaming strings, tool app logo (top left of green bar), and MacOS program transfer gif

Related: https://microbit-global.monday.com/boards/1550536443/pulses/1643568195

After discussion we've also tried for more clarity around the role of the micro:bits and the names of the programs we're downloading to them. The details of this are still subject to feedback but we'll merge for now.

### CODE REVIEW

File: src/components/ManualFlashingDialog.tsx
Comment (microbit-grace) [inline src/components/ManualFlashingDialog.tsx:1]: Require changing Windows and ChromeOS transfer program gifs as well

File: src/components/DefaultPageLayout.tsx [hunk 1]
```diff
@@ -62,7 +62,10 @@ const DefaultPageLayout = ({
   const toast = useToast();
 
   useEffect(() => {
-    document.title = intl.formatMessage({ id: titleId });
+    const appName = `micro:bit ${TOOL_NAME}`;
```
Comment (microbit-grace) [inline src/components/DefaultPageLayout.tsx:65]: Assumes we're not translating "micro:bit AI creator"

File: src/components/DefaultPageLayout.tsx [hunk 2]
```diff
@@ -62,7 +62,10 @@ const DefaultPageLayout = ({
...
+    document.title = titleId
+      ? `${intl.formatMessage({ id: titleId })} | ${appName}`
+      : appName;
```
Comment (microbit-grace) [inline src/components/DefaultPageLayout.tsx:68]: We somehow lost the logic where pages in the tool have titles "${name of page} | micro:bit AI creator" e.g. "Data samples | micro:bit AI creator".

File: src/components/ManualFlashingDialog.tsx
Comment (microbit-matt-hillsdon) [reply src/components/ManualFlashingDialog.tsx:1]: Let's track these separately and do them once the naming is final.

Comment (microbit-grace) [reply src/components/ManualFlashingDialog.tsx:1]: Added a subitem to the Monday item: https://microbit-global.monday.com/boards/1550536443/pulses/1643567124

File: index.html [hunk 3]
```diff
@@ -3,32 +3,38 @@
...
+    <title>micro:bit AI creator</title>
+    <meta property="og:title" content="micro:bit AI creator" />
     <link rel="canonical" href="%VITE_FULL_URL%" />
     <meta property="og:url" content="%VITE_FULL_URL%" />
     <meta name="twitter:card" content="summary_large_image" />
     <meta name="twitter:site" content="@microbit_edu" />
     <meta
       name="description"
-      content="Use our free machine learning tool as part of BBC micro:bit - the next gen's playground survey investigation. Explore machine learning, by training a computer model with your own data, then testing it to see if it works." />
+      content="Use our free micro:bit AI creator. Explore machine learning, by training a computer model with your own data, then testing it to see if it works."
```
Comment (microbit-matt-hillsdon) [inline index.html:20]: At some point we should redo this and other social metadata and consider capturing the MakeCode aspect.

File: lang/ui.en.json [hunk 4]
```diff
@@ -919,30 +915,6 @@
     "defaultMessage": "This browser is not supported",
     "description": ""
   },
-  "popup.outdatedmicrobit.button.later": {
```
Comment (microbit-matt-hillsdon) [inline lang/ui.en.json:922]: These were all unused

**Pull request merged**